frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -452,12 +453,18 @@
    "fieldname": "prepared_report_section",
    "fieldtype": "Section Break",
    "label": "Prepared Report"
+  },
+  {
+   "default": "0",
+   "fieldname": "currency_name_instead_of_symbol",
+   "fieldtype": "Check",
+   "label": "Currency name instead of symbol"
```

Comment:
I think this should rather be a "Select" field "Currency Indicator" with options "$ 10" and "10 USD". This way it's more intuitive to understand.

Reply:
I agree and changed it.